src/components/DDUtils.ts

```diff
@@ -1,39 +1,34 @@
-import { createTheme } from '@material-ui/core/styles'
-import { UserType } from '../state/dataTypes'
-import { DEFAULT_USER } from './constants'
+import { createTheme } from '@material-ui/core/styles';
+import { UserType } from '../state/dataTypes';
+import { DEFAULT_USER } from './constants';
 
 // utility functions
 export default class DDUtils {
-    static getProjectTitle() {
-        const user: UserType = DEFAULT_USER
+    static getProjectTitle(): string {
+        const user: UserType = DEFAULT_USER;
         if (user && user.displayName) {
-            return user.displayName.split(' ')[0] + "'s ProjectStyles"
+            return user.displayName.split(' ')[0] + "'s ProjectStyles";
         } else {
-            return 'Anonymous ProjectStyles'
+            return 'Anonymous ProjectStyles';
         }
     }
 
     static getProjectTheme() {
-        let theme
-        window.matchMedia &&
-        window.matchMedia('(prefers-color-scheme: dark)').matches
-            ? (theme = createTheme({ palette: { type: 'dark' } }))
-            : (theme = createTheme({ palette: { type: 'light' } }))
-        return theme
+        let theme;
+        // window.matchMedia &&
+        // window.matchMedia('(prefers-color-scheme: dark)').matches
+        //     ? (theme = createTheme({ palette: { type: 'dark' } }))
+        //     : (theme = createTheme({ palette: { type: 'light' } }));
+        theme = createTheme({ palette: { type: 'light' } });
+        return theme;
     }
 
-    static getDisplayName() {
-        const user: UserType = DEFAULT_USER
+    static getDisplayName(): string {
+        const user: UserType = DEFAULT_USER;
         if (user && user.displayName) {
-            return user.displayName.split(' ')[0]
+            return user.displayName.split(' ')[0];
         } else {
-            return 'Sign In'
+            return 'Sign In';
         }
     }
-
-    static validateEmail(email: string): boolean {
```

Comment:
why did you delete this?

Reply:
we're not using it anywhere. Material UI components have built in email validation.